XenAdmin/ConsoleView/VNCTabView.cs

```diff
@@ -1099,11 +1093,11 @@ private void OnDetectRDP_()
             {
                 log.DebugFormat("RDP detected for VM '{0}'", source == null ? "unknown/null" : source.name_label);
                 toggleToXVNCorRDP = RDP;
-
+                

```

Comment:
Nit: Trailing whitespace.

Reply:
That's the default behaviour from VS2022, i.e. if you add an empty line, it'll add whitespace that matches the indentation level